client/src/styles/main.less

```diff
@@ -18,4 +20,30 @@
   flex-direction: row;
 }
 
-.main { flex: 1; }
+.main {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+button {
+  box-shadow: none;
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
```

Comment:
No need to vendor prefix since postcss preprocessor has autoprefixer configured (webpack config)